package.json

```diff
@@ -33,6 +33,8 @@
     "should": "3.3.2",
     "xunit-file": "0.0.5"
   },
+  "preferGlobal": true,
```

Comment:
Indeed, the `preferGlobal` flag will only alert the user he *should* install it globally, but it will not cause any failure. ref: https://docs.npmjs.com/files/package.json#preferglobal